production/helm/loki/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Entries should include a reference to the pull request that introduced the chang
 
 [//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries bellow this line.)
 
+## 5.6.3
```

Comment:
Someone else already took 5.6.3. Please make it 5.6.4.

Reply:
Fixed.